components/button/index.css

```diff
@@ -106,6 +104,258 @@ governing permissions and limitations under the License.
   --spectrum-button-intended-icon-size: var(--spectrum-workflow-icon-size-300);
 }
 
+/* Variant colors */
+.spectrum-Button {
+  &.spectrum-Button--accent,
+  &.spectrum-Button--negative {
+    --spectrum-button-content-color-default: var(--spectrum-white);
+    --spectrum-button-content-color-hover: var(--spectrum-white);
+    --spectrum-button-content-color-down: var(--spectrum-white);
+    --spectrum-button-content-color-focus: var(--spectrum-white);
+  }
+
+  &.spectrum-Button--outline {
+    --spectrum-button-background-color-disabled: transparent;
+    --spectrum-button-border-color-disabled: var(--spectrum-disabled-border-color);
+    --spectrum-button-content-color-disabled: var(--spectrum-disabled-content-color);
+  }
+
+  /* ---- Accent ---- */
+  &.spectrum-Button--accent {
+    --spectrum-button-background-color-default: var(--spectrum-accent-background-color-default);
+    --spectrum-button-background-color-hover: var(--spectrum-accent-background-color-hover);
+    --spectrum-button-background-color-down: var(--spectrum-accent-background-color-down);
+    --spectrum-button-background-color-focus: var(--spectrum-accent-background-color-key-focus);
+
+    &.spectrum-Button--outline {
```

Comment:
I'm having trouble finding the S2 spec for Accent Outline, could you point me to that?

Reply:
Good callout on this one! The token values are left as they were previously, but do not appear to be on the specs Figma. I'll ask the design team and see if that needs to be added to the Figma and if any of those token values have changed.

Reply:
Per design, the outline style has been deprecated for the accent and negative variants. I've removed those styles. I will reflect this better in the controls once PR #2578 gets merged in, as that makes big changes to the structure of the stories.